docs/src/main/paradox/index.md

```diff
@@ -15,7 +15,7 @@ The Akka Persistence R2DBC plugin allows for using SQL database with R2DBC as a
 * [PostgreSQL JSON](postgres_json.md)
 * [Projection](projection.md)
 * [Configuration](config.md)
-* [Data partitioning](data-partition.md)
+* [Database sharding](data-partition.md)
```

Comment:
Also change the name of the file for links? Or too much hassle with redirecting?

Reply:
was also thinking about it, but I don't see it as very important